src/main/scala/cromwell/binding/package.scala

```diff
@@ -25,7 +25,7 @@ package object binding {
   type FullyQualifiedName = String
   type LocallyQualifiedName = String
   type CallInputs = Map[String, WdlValue]
-  case class CallOutput(wdlValue: WdlValue, hash: SymbolHash)
+  case class CallOutput(wdlValue: WdlValue, hash: Option[SymbolHash])
```

Comment:
Comment definitely noted. I'm just waiting for someone else to start using unions before I tell anyone else "copy that".